core/templates/components/shared-component.module.ts

```diff
@@ -123,6 +124,7 @@ const firebaseAuthProviders = AuthService.firebaseAuthIsEnabled ? [
     SharingLinksComponent,
     SkillSelectorComponent,
     SkillMasteryViewerComponent,
+    SkillDescriptionEditorComponent,
```

Comment:
Shouldn't this go in the skill-editor-page module?

Reply:
Moved them